concepticondata/concepticon.tsv

```diff
@@ -1190,7 +1190,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 1189	ULCER	The body	An open sore of the skin, eyes or mucous membrane, often caused by an initial abrasion and generally maintained by an inflammation and/or an infection.	Person/Thing	
 1190	HARE	Animals	Mammal of the family hares and rabbits (Leporidae) with long ears, short tail and hindlegs which are shorter than the forelegs and permit running quickly.	Person/Thing	
 1191	COOKED RICE	Food and drink	Boiled rice.	Person/Thing	
-1192	THREATEN	Speech and language	To put in danger.	Action/Process	
+1192	THREATEN	Speech and language	To utter intentions of injury or punishment against. [WN:1192]	Action/Process	
```

Comment:
Not necessarily `intentions`, but simply (as implied by the gloss), `threats`. I'd say "To make a threat of injury or punishment against someone", but please check if the meaning in the wordlist that use it is not something like "menace" (e.g., "the beast threatens us").